server/Dockerfile

```diff
@@ -4,6 +4,7 @@ ENV KEYCLOAK_VERSION 4.3.0.Final
 ENV JDBC_POSTGRES_VERSION 42.2.2
 ENV JDBC_MYSQL_VERSION 5.1.46
 ENV JDBC_MARIADB_VERSION 2.2.3
+ENV JDBC_MSSQL_VERSION 6.4.0
```

Comment:
If I am not mistaken, downloading the JDBC driver needs to be added to build-keycloak.sh